media-libs/libyuv/libyuv-9999.ebuild

```diff
@@ -14,10 +14,8 @@ if [[ ${PV} == "9999" ]] ; then
 else
 	# to diff against upstream (apparently not stable):
 	# https://chromium.googlesource.com/libyuv/libyuv.git/+archive/${commit}.tar.gz
-	MYTAG="0.0.1904.20250204"
-	SRC_URI="https://salsa.debian.org/debian/libyuv/-/archive/upstream/${MYTAG}/libyuv-upstream-${MYTAG}.tar.bz2 -> ${P}.tar.bz2"
-	S="${WORKDIR}/libyuv-upstream-${MYTAG}"
-	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/N-R-K/stable-tarball-host/releases/download/0/libyuv-${PV}.tar.bz2"
```

Comment:
What's an example value of `${commit}`? https://chromium.googlesource.com/libyuv/libyuv.git/+archive/libyuv-1909.tar.gz doesn't work for me.

Reply:
`commit` is literal commit hash. They don't tag their releases. This version in specific was added with commit [9f9b5cf660dcfa0d3fdee41cf4ffbe4bb6e95114](https://chromium.googlesource.com/libyuv/libyuv/+/9f9b5cf660dcfa0d3fdee41cf4ffbe4bb6e95114%5E%21).